lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -126,6 +126,11 @@
             be set in the container being built.
           - Needs Docker SDK for Python >= 3.7.0.
         type: bool
+      target:
+        description:
+          - When building an image specifies an intermediate build stage by
+            name as a final stage for the resulting image.
+        type: str
```

Comment:
You need to add `version_added: "2.9"`. Also, you need a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to).